src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -254,25 +246,44 @@ export class QueryEditor extends BaseEditor {
 		return this._listDatabasesActionItem;
 	}
 
-	private setTaskbarContent(): void {
-		// Create HTML Elements for the taskbar
+	private setTaskbarContent(input: QueryEditorInput): void {
+		// Remove current actions from the taskbar
+		while (this.taskbar.length() > 0) {
```

Comment:
setContent should be clearing the existing items so I'm not sure why this is necessary - were you seeing some issue here?